packages/babel-plugin-transform-react-jsx-source/test/fixtures/react-source/with-source/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "nodePlatform": ["linux", "darwin"]
```

Comment:
WDYT about calling it "os"?

Reply:
I am okay with `os` as long as we clear document that this value is one of
`'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', 'win32'`. I did mistyped `win32` to `windows` anyway.

Or we may align to travis-ci to have `linux`, `macos`, `windows` only. Under the hood we map them to `process.platform`.

Reply:
I think that it's ok to just use what `process.platform` uses, and document it.

Reply:
Another thoughts: could we foresee that `babel` itself would not use an option named `os`? Otherwise it will be a potential conflict.

Reply:
Well, we never had a feature request for it and I can't imagine how it why it would be necessary.
I proposed `os` because it is not related to node, but also `platform` would work (even if `os` is still the most descriptive)

Reply:
Since this is only for our internal fixtures, we can always change it? Up to you

Reply:
Then it's consensus: use `os` with candidate values from `process.platform`.